packages/core/src/toggle-button-group/ToggleButtonGroup.css

```diff
@@ -1,11 +1,11 @@
 .saltToggleButtonGroup {
   display: flex;
   background: var(--salt-container-primary-background);
-  border: var(--salt-size-border) var(--salt-container-borderStyle) var(--salt-container-primary-borderColor);
+  border: var(--salt-size-fixed-100) var(--salt-container-borderStyle) var(--salt-container-primary-borderColor);
   border-radius: var(--salt-palette-corner-weak, 0);
   width: fit-content;
   gap: var(--salt-spacing-50);
-  padding: calc(var(--salt-spacing-50) - var(--salt-size-border));
+  padding: calc(var(--salt-spacing-50) - var(--salt-size-fixed-100));
```

Comment:
as this is padding should it be spacing-fixed instead of size-fixed?

Reply:
This is an "inside border" calculation so it's referring to the width of the border